simcon_project/conversation_templates/models/template_node_choice.py

```diff
@@ -6,6 +6,6 @@ class TemplateNodeChoice (models.Model):
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     choice_text = models.CharField(max_length=500)
     destination_node = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='parent_choices', on_delete=models.DO_NOTHING)
```

Comment:
I think we need to set blank=True, null=True here to allow for empty values that get set to NULL automatically. This makes it so terminal nodes (or any node) can have a choice that does not have a chosen destination without Django throwing the IntegrityError. That keeps those choices from getting saved in the backend.